src/components/modifyQuantity/ModifyQuantity.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, ViewStyle } from 'react-native';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+import type { IconName } from '../icons/IconProps';
+import { TextInput } from '../inputs/TextInput';
+import { Body } from '../typography/Body';
+
+interface Props {
+    text: string;
+    onAdd: () => void;
+    minValue: number;
+    maxValue: number;
+    onMinus: () => void;
+    style?: ViewStyle;
+    inputValue: number;
+    icon?: IconName;
+}
+
+export const ModifyQuantity = ({
+    text,
+    onAdd,
+    onMinus,
+    style,
+    inputValue,
+    minValue,
+    maxValue,
+    icon = 'etiquette',
+}: Props) => {
+    const theme = useTheme();
+
+    const minusDisabled = minValue >= inputValue;
+    const addDisabled = maxValue <= inputValue;
+
+    const styles = StyleSheet.create({
+        container: {
+            width: '100%',
+            flexDirection: 'row',
+            alignItems: 'center',
+            padding: theme.sw.spacing.l,
+            backgroundColor: theme.sw.colors.neutral[50],
+            borderRadius: 24,
+            justifyContent: 'space-between',
+            elevation: 1,
+            ...style,
+        },
+        text: {
+            marginLeft: theme.sw.spacing.s,
+            color: theme.sw.colors.neutral[800],
+        },
+        iconContainer: {
+            flexDirection: 'row',
+            alignItems: 'center',
+        },
+        inputContainer: {
+            alignItems: 'center',
+            flexDirection: 'row',
+        },
+        input: {
+            marginBottom: 0,
+            marginHorizontal: theme.sw.spacing.l,
+        },
+        inputText: {
+            fontSize: 28,
```

Comment:
I see 16px on the design system no ?
cf: https://www.figma.com/file/MD0xdHA7E93GKAwkSRpduK/SW-mobile-app---Design-Kit?node-id=984-1415&t=3kNwwPqELMX05L3S-4